test/plugins/CMakeLists.txt

```diff
@@ -1,17 +1,24 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
-if (CMAKE_SYSTEM_NAME MATCHES "Linux")
-  add_subdirectory(wasmedge_process)
+if(WASMEDGE_PLUGIN_PROCESS)
+  if (CMAKE_SYSTEM_NAME MATCHES "Linux")
+    add_subdirectory(wasmedge_process)
+  endif()
```

Comment:
We may need to add a warning message if users want to enable the process plugin on the other platforms.